### PR TITLE
var-som-mx8m-nano support

### PR DESCRIPTION
**Pull request by cbrake**

### CODE REVIEW

**Comment by cbrake** [comment]
manually baking `imx-boot` gets me past this issue, and now have an image building. Need to create a patch for u-boot, yet and should then be ready to try it.

File: sources/meta-yoe/recipes-support/updater/files/imx8mn-var-som/platform [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+BOOT_MOUNT_POINT=/boot
+ROOTFS_MOUNT_POINT=/root
+DATA_MOUNT_POINT=/data
+USB_MOUNT_POINT=/media/usb
+SD_MOUNT_POINT=/media/sd
+
+# Size in 'M' for MB, 'G' for GB
+RESERVED_SIZE=2M
+BOOT_SIZE=100M
+ROOTFS_SIZE=1G
```
Comment (kraj) [inline sources/meta-yoe/recipes-support/updater/files/imx8mn-var-som/platform:54]: make it 2GB

Comment (cbrake) [reply sources/meta-yoe/recipes-support/updater/files/imx8mn-var-som/platform]: done, thanks!

**Comment by cbrake** [comment]
@kraj  It seems imx-boot is not getting rebuilt when u-boot changes, and I think this is what gets stuck in the SD raw device for bootloaders. Just FYI, not a huge deal.

**Comment by cbrake** [comment]
this platform is a bit different than the others -- seems to have 3 SD controllers:

```
~ # dmesg | grep mmc
[    0.000000] Kernel command line: console=ttymxc3,115200 earlycon=ec_imx6q,0x30a60000,115200 root=/dev/mmcblk1p2 rootwait rw cma=576M cma_name=linux,cma
[    1.240837] mmc2: SDHCI controller on 30b60000.mmc [30b60000.mmc] using ADMA
[    1.241122] mmc0: SDHCI controller on 30b40000.mmc [30b40000.mmc] using ADMA
[    1.401764] mmc2: new HS400 Enhanced strobe MMC card at address 0001
[    1.409014] mmcblk2: mmc2:0001 AJTD4R 14.6 GiB 
[    1.416872] mmcblk2boot0: mmc2:0001 AJTD4R 4.00 MiB 
[    1.428212] mmcblk2boot1: mmc2:0001 AJTD4R 4.00 MiB 
[    1.441851] mmcblk2rpmb: mmc2:0001 AJTD4R 4.00 MiB, chardev (234:0)
[    2.036285] sdhci-esdhc-imx 30b50000.mmc: Got CD GPIO
[    2.078024] mmc1: SDHCI controller on 30b50000.mmc [30b50000.mmc] using ADMA
[    2.279659] mmc1: host does not support reading read-only switch, assuming write-enable
[    2.328170] mmc1: new ultra high speed SDR104 SDHC card at address aaaa
[    2.337396] mmcblk1: mmc1:aaaa SC32G 29.7 GiB 
[    2.379881]  mmcblk1: p1
[    6.420135] EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null). Quota mode: none.
```

I'm not sure what the first one is -- perhaps wifi ...

**Comment by cbrake** [comment]
@kraj this is ready for review/merge

**Review by kraj — APPROVED** [review]
it will be good to add changelog entry for adding new project. Otherwise it looks ok.

**Pull request merged**